<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Puzzle</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="gameWindow" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="newGameAction"/>
    <addaction name="resetAction"/>
    <addaction name="pauseAction"/>
    <addaction name="hintAction"/>
    <addaction name="solutionAction"/>
    <addaction name="separator"/>
    <addaction name="saveGameAction"/>
    <addaction name="loadGameAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="saveGameAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="newGameAction">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="hintAction">
   <property name="text">
    <string>Hint</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="solutionAction">
   <property name="text">
    <string>Solution</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="loadGameAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
